test/core/simd/simd_lane.wast

```diff
@@ -489,14 +489,8 @@
   (v128.const i8x16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1 0) (f32.const 4.0)))) "type mismatch")
 
 ;; v8x16.shuffle: the 1st argument must be 16-byte literals in 0..32
-(assert_invalid (module (func (result v128)
-  (v8x16.shuffle 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14
-  (v128.const i8x16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1 0)
-  (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15)))) "invalid lane length")
-(assert_invalid (module (func (result v128)
-  (v8x16.shuffle 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16
-  (v128.const i8x16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1 0)
-  (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15)))) "invalid lane length")
+(assert_malformed (module quote "(func v8x16.shuffle 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14)") "invalid lane length")
+(assert_malformed (module quote "(func v8x16.shuffle 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16)") "invalid lane length")
```

Comment:
Would this work like this without arguments?

Reply:
Hm sorry I'm not sure I quite understand, could you clarify what you're thinking should work without arguments?

Reply:
Does the shuffle instruction need all arguments for this test to work correctly? You are not providing the two v128 arguments to it.

Reply:
Ah ok, I see what you mean, and sure! I've updated the test to typecheck but still have this expected assertion failure.

Sorry I also don't know how to have multiline strings in the wat format so it's a bit messy...